plugins/UsersManager/Controller.php

```diff
@@ -743,4 +744,11 @@ private function getIgnoreCookieSalt()
     {
         return md5(SettingsPiwik::getSalt());
     }
+
+    public static function getUserEditExtensions(): array
```

Comment:
Couldn't find any reason for the method to be public and static, unless I missed something?

```suggestion
    private function getUserEditExtensions(): array
```

Reply:
If we refactor to obtain the extra components from Vue, we might be able to remove this.